profiles/base.yaml

```diff
@@ -999,6 +999,7 @@ skeleton_key: skeleton key
 # Set to false to use lockpicks instead.
 use_skeleton_key: false
 use_lockpick_ring: true
+balance_lockpick_container: false
```

Comment:
Other new properties that are only used the new pick were nested under "pick:", i.e.
```
pick:
  balance_lockpick_container: false
```
Obviously not a 'must change', just something to consider.